From: Dimitri John Ledkov <xnox@ubuntu.com>
Date: Wed, 25 May 2022 18:22:01 +0800
Subject: * Stop setting LB_INITRAMFS_COMPRESSION default,
 and instead fallback to

    using initramfs-tools default. LB_INITRAMFS_COMPRESSION is now only to
    override whatever initramfs-tools' default compression is. This thus
    makes live-build default to lz4. LP: #1831736
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1831736
---
 functions/defaults.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/functions/defaults.sh b/functions/defaults.sh
index 5718f07..713190b 100755
--- a/functions/defaults.sh
+++ b/functions/defaults.sh
@@ -220,8 +220,6 @@ Set_defaults ()
 			;;
 	esac
 
-	LB_INITRAMFS_COMPRESSION="${LB_INITRAMFS_COMPRESSION:-gzip}"
-
 	# Setting initsystem
 	case "${LB_MODE}" in
 		ubuntu|kubuntu)
